src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -296,11 +343,25 @@ class ShareView extends React.PureComponent {
                       setValue={this.setHideNavigation}
                     />
                     <PaddedPreference
+                      title="Single file entry script"
```

Comment:
and actually it should be just like github *(not URL-encoded)*:

https://github.com/faceyspacey/redux-first-router/blob/master/src/connectRoutes.js

see how ur app's file structure becomes a standard part of the URL. **THAT IS THE CORRECT WAY!**